sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Release History
 
-## 5.2.0-preview.2 (Unreleased)
+## 5.2.0-preview.2 (2020-08-10)
 
+### Acknowledgments
+
+Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:
+
+- Daniel Marbach _([GitHub](https://github.com/danielmarbach))_
+
+### Changes
+
+#### Processing events
+
+- Load balancing will now detect when it has reached a balanced state more accurately; this will allow it to operate more efficiently when `LoadBalancingStrategy.Greedy` is in use.
+
+- The approach used for creation of checkpoints has been updated to interact with Azure Blob storage more efficiently.  This will yield major performance improvements when soft delete was enabled and minor improvements otherwise.
+
+- Load balancing now has better recognition for being in a recovery state and will aggressively reclaim partitions for it is the recognized owner when the current instance did not perform the original ownership claim.  Previously, those partitions were redistributed on a 1-by-1 basis as part of the standard cycle. 
```

Comment:
"for it is" sentence mangling.

Reply:
I seem to have forgotten a word.   Thanks!